flight/PiOS/Common/pios_hal.c

```diff
@@ -48,6 +47,18 @@
 #include <openlrs.h>
 #endif
 
+#ifdef PIOS_INCLUDE_USART
+#include <pios_usart_priv.h>
```

Comment:
I will move the ones I can into public headers.  Where we get into some trouble is when something like PPM is target specific and in stm32/inc or whatever-- but PIOS_HAL needs the type in its function signature.